File: official/docs/csharp/billing/AddFund.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task AddFunds()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        bool success = await Billing.FundWallet("2000", PaymentMethod.Priority.Primary);

        new TestOutputHelper().WriteLine(success.ToString());
    }
}

File: official/docs/java/billing/AddFund.java
package billing;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.Billing;
import com.easypost.model.PaymentMethod;

public class AddFund {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        boolean success = Billing.fundWallet("2000", PaymentMethod.Priority.PRIMARY);

        System.out.println(success);
    }
}
